packages/components/src/tag/docs/tag/variants.tsx

```diff
@@ -0,0 +1,46 @@
+import { TagGroup, Tag, TagList } from "@hopper-ui/components";
+
+export default function Example() {
+    return (
```

Comment:
it's a detail but can you add `<Div margin="stack-lg" width="80%">` to make the presentation more harmonious.